frontend/src/utils/fieldValidation.ts

```diff
@@ -250,22 +250,21 @@ export const createDecimalValidationRules: ValidationRuleFn<
       if (!validateByValue) return true
 
       if (
-        customMin &&
-        customMax &&
+        customMin !== null &&
+        customMax !== null &&
         (numVal < customMin || numVal > customMax)
       ) {
         return `Please enter a decimal between ${formatNumberToLocaleString(
           customMin,
         )} and ${formatNumberToLocaleString(customMax)} (inclusive)`
       }
-
-      if (customMin && numVal < customMin) {
+      if (customMin !== null && numVal < customMin) {
```

Comment:
supernit: what happened to the whitespace HAHAHA

Reply:
what whitespace